google/cloud/forseti/services/scanner/dao.py

```diff
@@ -40,14 +40,17 @@ def define_violation(model_name, dbengine):
     """
 
     base = declarative_base()
-    violations_tablename = '{}_violations'.format(model_name)
+    # pylint: disable=too-many-format-args
```

Comment:
Remove the pylint disable and remove the .format() from the string. The tablename is just 'violations'

Reply:
Done.